project.yaml

```diff
@@ -938,23 +940,40 @@ Layer:
     <<: *extents
     Datasource:
       <<: *osm2pgsql
+      # This is one of the most complex layers, so it bears explaining in some detail
+      # It is necessary to
+      # - Have roads and railways in the same layer to get ordering right
+      # - Return two linestrings for ways which are both a road and railway
+      # - Compute z_order here, rather than rely on osm2pgsql z_order
       table: |-
```

Comment:
And I think that it would be a good idea to mention in this comment that aeroways are also handled here.

It may be removed if/once #2169 is done.